.changeset/loud-walls-accept.md

```diff
@@ -0,0 +1,5 @@
+---
+'next-urql': major
+---
+
+Expose initUrqlClient function in order to run it in SSR mode
```

Comment:
```suggestion
Expose `initUrqlClient` function so that a `Client` can be created manually for use in Next's newer SSR methods manually, such as `getServerSideProps`
```

Just noting here that this is for the changelog, so it'll need to be as clear as possible; also there are two other changeset files that'll need to be delete 😅

Reply:
Your suggestion has been applied :)